app.js

```diff
@@ -261,16 +302,20 @@ class WitnessApp extends LitElement {
           ${this.csrftoken ? html`
             <input type="hidden" name="${this.csrftoken_name}" value="${this.csrftoken}"/>` : ``}
           <div class="field">
-          <label class="label">URLs</label>
+            <label for="urls" class="label">URLs</label>
             <div class="control">
-              <textarea id="urls" rows="3" required class="textarea is-link" placeholder="Enter one or more URLs on each line"></textarea>
+              <textarea id="urls" rows="3" required class="textarea ${this.submittedUrlsInvalid ? "is-danger": ""}"
+                        placeholder="Enter one or more URLs on each line"
+                        aria-invalid="${this.submittedUrlsInvalid ? "true": "false"}"
+                        aria-describedby="urls-errors"></textarea>
             </div>
+            ${this.submittedUrlsInvalid ? html`<p id="urls-errors" class="help is-danger">${this.fieldErrorMessage}</p>` : ``}
           </div>
 
           <div class="field">
-            <label class="label">Label (Optional)</label>
+            <label for="tag" class="label">Label (Optional)</label>
             <div class="control">
-              <input id="tag" type="text" class="input" value="My Archive" placeholder="My Archive"/>
+              <input id="tag" type="text" class="input" value="" placeholder="My Collection"/>
```

Comment:
Its set as default as well as placeholder so that default "Label" is something nice, not the job id..
I guess maybe should just have the label default to blank

Reply:
Ah, gotcha, I think I was thrown by the "(Optional)" in the label: I figured you were going quick and intended "My Archive" as a placeholder, not both a placeholder and a default value. I also didn't realize I left my "Collection" experiment in there.... I realize that "Collection" is a term-of-art in WR land for the results of a single capturing process, which is why you had "Archive".... which is a term-of-art in Perma land for the file containing the results of a single capturing process :-). I committed that change by accident; I wasn't aiming to weigh in on nomenclature.